new_project/examples/separate_apps/src/maker.ts

```diff
@@ -0,0 +1,84 @@
+import { formatEther } from "ethers/utils";
+import readLineSync from "readline-sync";
+import { toBitcoin } from "satoshi-bitcoin-ts";
+import { checkEnvFile, startClient } from "./lib";
+
+(async function main() {
+    checkEnvFile(process.env.DOTENV_CONFIG_PATH!);
+
+    const maker = await startClient(0);
+
+    console.log(
+        "[Maker] Bitcoin balance: %f. Ether balance: %f",
+        parseFloat(await maker.bitcoinWallet.getBalance()).toFixed(2),
+        parseFloat(
+            formatEther(await maker.ethereumWallet.getBalance())
+        ).toFixed(2)
+    );
+
+    const peerId = await maker.comitClient.getPeerId();
+    const addressHint = await maker.comitClient
+        .getPeerListenAddresses()
+        .then(addresses => addresses[0]);
+
+    console.log("[Maker] peerId:", peerId);
+    console.log("[Maker] addressHint:", addressHint);
+
+    // Note that we assume the maker published an offer
+    // somehow somewhere and that someone (a taker)
+    // will take such offer by send a SWAP request to the
+    // maker's COMIT node. This initial phase is not done
+    // as part of this example.
```

Comment:
:upside_down_face: I know what you mean, but, having mentioned that the taker will take the offer by sending the swap request, it seems that now you're saying that sending a swap request is also not part of the example. I would either remove this last sentence or explicitly say: "Publishing and finding offers is not part of this example".